packages/ui/src/breadcrumbs.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/display-name */
+import { Box, PropsWithBox, Text } from "@saleor/macaw-ui/next";
+import Link from "next/link";
+import React from "react";
+import styles from "./breadcrumbs.module.css";
+
+export type BreadcrumbsItemProps = PropsWithBox<{
+  href?: string;
+  active?: boolean;
+  separator?: React.ReactNode;
+}>;
+
+const BreadcrumbsItem = ({
+  children,
+  href,
+  separator = ">",
+  active = false,
+  ...p
+}: BreadcrumbsItemProps) => {
+  return (
+    <Box as="li" {...p}>
+      <Text variant="title" color={active ? "textNeutralDefault" : "textNeutralSubdued"}>
+        {href ? (
+          <Link className={styles.link} href={href}>
+            {children}
+          </Link>
+        ) : (
+          children
+        )}
+        {!active && (
```

Comment:
I thnk separator should be in container/parent. Separator is not about "active", but about being *last*

So, parent "knows" what is last 

Also if we have props "active" that will overwrite (make middle item active), entire layout will break, we shouldn't allow that